e2e/Makefile

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
💯 for using Make!!

Reply:
I think we should include this test execution into the github action. Also creating a Makefile at the root dir, including both test executions, would be great

Reply:
Done.